tests/python/gaia-ui-tests/gaiatest/apps/system/regions/activities.py

```diff
@@ -53,6 +53,8 @@ def tap_camera(self):
         camera = Camera(self.marionette)
         self.wait_for_condition(lambda m: self.apps.displayed_app.name == camera.name)
         self.apps.switch_to_displayed_app()
+        camera.wait_for_loading_spinner_displayed()
+        camera.wait_for_loading_spinner_hidden()
```

Comment:
I tried to do that, but I'm getting intermittent failures of other camera tests that are using this method. As I noticed, sometimes the spinner is displayed just for ~1 sec or not at all in tests that are launching camera from Homescreen.
Though, while launching it from another app, like Messages in our case, it takes more time to load, so the loading spinner is displayed for more time. 
So I'd rather use this methods independently, and not adding them in <code>wait_for_capture_ready()</code>